src/Files.Uwp/ViewModels/SelectedItemsPropertiesViewModel.cs

```diff
@@ -192,6 +217,14 @@ public bool ItemSizeProgressVisibility
             set => SetProperty(ref itemSizeProgressVisibility, value);
         }
 
+        private bool uncompresseditemSizeProgressVisibility = false;
+
+        public bool UncompressedItemSizeProgressVisibility
```

Comment:
```suggestion
        public bool IsUncompressedItemSizeProgressVisibile
```
This might work better since the property is a boolean.